classes/class-wc-connect-note-dhl-live-rates-available.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * WooCommerce Shipping note: DHL live rates available.
+ *
+ * Only for legacy customers that had the feature available.
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+if ( ! class_exists( 'WC_Connect_Note_Dhl_Live_Rates_Available' ) ) {
```

Comment:
```suggestion
```
I know these checks are added everywhere in the code but they are not really needed provided you include the file with `requrie_once`

Reply:
Removed.